pom.xml

```diff
@@ -23,7 +23,7 @@
         <version>1.0.5</version>
     </parent>
     <modelVersion>4.0.0</modelVersion>
-    <groupId>jakarta.tutorial</groupId>
+    <groupId>ee.jakarta.tutorial</groupId>
```

Comment:
This does not need to change. The **groupId** can remain `jakarta.tutorial`.

Reply:
It would be technically ok, but I think it's better not to pollute the `jakarta` namespace, specially given the tutorial is still so GlassFish based.

Anyway, I can revert it to `jakarta.tutorial` if that's what people prefer.